policies/github/repository.rego

```diff
@@ -131,13 +163,13 @@ missing_default_branch_protection := false {
 # description: The history of the default branch is not protected against deletion for this repository.
 # custom:
 #   remediationSteps:
-#     - Note: The remediation steps applys to legacy branch protections, rules set based protection should be updated from the rules set page
-#     - Make sure you have admin permissions
-#     - Go to the repo's settings page
-#     - Enter "Branches" tab
-#     - Under "Branch protection rules"
-#     - Click "Edit" on the default branch rule
-#     - Uncheck "Allow deletions", Click "Save changes"
+#     - Note: The remediation steps apply to legacy branch protections, rules set-based protection should be updated from the rules set page
```

Comment:
There's a problem with this option, without quotes this line won't get rendered (just now realized this problem also existed before)